### PR TITLE
Removed flags to EGL for compatibility with new computers

### PR DESCRIPTION
**Pull request by marcusnaslund**
This change is needed to run on the new laptops.

I tried including the first part and running with only the second part removed, and that worked too. So maybe that's enough?

### CODE REVIEW

**Comment by marcusnaslund** [comment]
Comments/feedback/criticism please.

File: source/draw/gpu/opengl/backend/egl/EglDisplayContext.ooc [hunk 1]
```diff
@@ -97,7 +87,6 @@ EglDisplayContext: class extends DisplayContext {
 			success &= eglGetConfigAttrib(this _eglDisplay, matchingConfigs[i], EGL_GREEN_SIZE, green&)
 			success &= eglGetConfigAttrib(this _eglDisplay, matchingConfigs[i], EGL_BLUE_SIZE, blue&)
 			success &= eglGetConfigAttrib(this _eglDisplay, matchingConfigs[i], EGL_ALPHA_SIZE, alpha&)
-
```
Comment (davidpiuva) [inline source/draw/gpu/opengl/backend/egl/EglDisplayContext.ooc]: This is not needed.

File: source/draw/gpu/opengl/backend/egl/EglDisplayContext.ooc [hunk 2]
```diff
@@ -35,21 +35,11 @@ EglDisplayContext: class extends DisplayContext {
...
-			EGL_SAMPLES, 0,
-			EGL_DEPTH_SIZE, 0,
-			EGL_BIND_TO_TEXTURE_RGBA, EGL_DONT_CARE,
 			EGL_NONE] as Int*
 		chosenConfig: Pointer = this _chooseConfig(configAttribs)
 		pbufferAttribs := [
 			EGL_WIDTH, 1,
 			EGL_HEIGHT, 1,
-			EGL_TEXTURE_TARGET, EGL_NO_TEXTURE,
-			EGL_TEXTURE_FORMAT, EGL_NO_TEXTURE,
```
Comment (davidpiuva) [inline source/draw/gpu/opengl/backend/egl/EglDisplayContext.ooc]: This was the last change before it worked on the laptop.

File: source/draw/gpu/opengl/backend/egl/EglDisplayContext.ooc [hunk 3]
```diff
@@ -35,21 +35,11 @@ EglDisplayContext: class extends DisplayContext {
...
 		configAttribs := [
 			EGL_SURFACE_TYPE, EGL_PBUFFER_BIT,
-			EGL_RENDERABLE_TYPE, EGL_OPENGL_ES2_BIT,
-			EGL_BLUE_SIZE, 8,
-			EGL_GREEN_SIZE, 8,
-			EGL_RED_SIZE, 8,
-			EGL_ALPHA_SIZE, 8,
-			EGL_SAMPLES, 0,
-			EGL_DEPTH_SIZE, 0,
-			EGL_BIND_TO_TEXTURE_RGBA, EGL_DONT_CARE,
```
Comment (davidpiuva) [inline source/draw/gpu/opengl/backend/egl/EglDisplayContext.ooc]: You can try if readding these still works on the laptop in order to minimize the risk of breaking other computers.

File: source/draw/gpu/opengl/backend/egl/EglDisplayContext.ooc [hunk 4]
```diff
@@ -35,21 +35,11 @@ EglDisplayContext: class extends DisplayContext {
 		This validate(eglBindAPI(EGL_OPENGL_ES_API), EGL_TRUE, "eglBindAPI")
 		configAttribs := [
 			EGL_SURFACE_TYPE, EGL_PBUFFER_BIT,
-			EGL_RENDERABLE_TYPE, EGL_OPENGL_ES2_BIT,
```
Comment (davidpiuva) [inline source/draw/gpu/opengl/backend/egl/EglDisplayContext.ooc]: EGL_OPENGL_ES2_BIT should remain if it works on the laptop.

File: source/draw/gpu/opengl/backend/egl/EglDisplayContext.ooc (hunk 2 above)
Comment (marcusnaslund) [inline source/draw/gpu/opengl/backend/egl/EglDisplayContext.ooc]: and it's been fixed separately, it had nothing to do with opengl.

**Comment by marcusnaslund** [comment]
Updated with less removals. This still runs on the computer.

**Comment by marcusnaslund** [comment]
Works on PRInt as well. :tada:

**Comment by marcusnaslund** [comment]
@thomasfanell or @sebastianbaginski please update the bridge lib

**Pull request merged**